streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -331,25 +340,29 @@
                                                                 Serde<T> aggValueSerde);
 
     /**
-     * Aggregate values of this stream by key on a window basis.
+     * Aggregate values of this stream by key on a window basis into a new instance of windowed {@link KTable}
+     * with default serializers and deserializers.
      *
-     * @param initializer   the class of {@link Initializer}
-     * @param aggregator    the class of {@link Aggregator}
+     * @param initializer   the instance of {@link Initializer}
+     * @param aggregator    the instance of {@link Aggregator}
      * @param windows       the specification of the aggregation {@link Windows}
-     * @param <T>           the value type of the aggregated table
+     * @param <T>           the value type of the resulted {@link KTable}
      */
     <T, W extends Window> KTable<Windowed<K>, T> aggregateByKey(Initializer<T> initializer,
                                                                 Aggregator<K, V, T> aggregator,
                                                                 Windows<W> windows);
 
     /**
-     * Aggregate values of this stream by key without a window basis, and hence
-     * return an ever updating table
+     * Aggregate values of this stream by key into a new instance of ever-updating {@link KTable}.
```

Comment:
Ah, I see the difference:  I suppose "ever-updating" refers to this method returning a a *non-windowed* KTable?  If that's right, then we should perhaps clarify this a bit in the docstring (it's easy to miss -- these methods all have similar names).